02-manpages.patch by Luis Bustamante <luferbu@fluidsignal.com>
Additional manpages from various authors. Fixed mtools section.

Signed-off-by: Luis Bustamante <luferbu@debian.org>
Modified-by: Anibal Monsalve Salazar <anibal@debian.org>
Signed-off-by: Anibal Monsalve Salazar <anibal@debian.org>

--- a/lz.1	1969-12-31 19:00:00.000000000 -0500
+++ b/lz.1	2003-01-19 14:24:39.000000000 -0500
@@ -0,0 +1,42 @@
+'\" t
+.\" ** The above line should force tbl to be a preprocessor **
+.\" Man page for lz
+.\"
+.\" Copyright (C), 2000, Robert A.R. King
+.\"
+.\" You may distribute under the terms of the GNU General Public
+.\" License as specified in the file COPYING that comes with the mtools
+.\" package
+.\"
+.\" Fri Dec  1 01:50:54 EST 2000 Robert A.R. King (Robert.King@mailbox.gu.edu.au)
+.\" 
+.TH LZ 1 "Wed Feb 23 00:00:00 EET 2000" "" "Mtools Users Manual"
+.SH NAME
+lz \- gunzips and shows a listing of a gzip'd tar'd archive
+.SH SYNOPSIS
+.\" The command line
+.B lz 
+.I file
+.SH DESCRIPTION
+.B lz
+provides a listing of a gzip'd tar'd archive, that is a \fBtar\fR(1) archive
+compressed with the \fBgzip\fR(1) utility.
+It is not strictly necessary on Debian GNU/Linux, because the GNU
+\fBtar\fR(1) program provides the same capability with the command
+
+.B tar -tzf
+.I file
+
+but this utility is provided in the mtools package for other platforms and 
+is retained here for completeness.
+
+.SH AUTHOR
+Robert King (Robert.King@mailbox.gu.edu.au) wrote this page for the
+.I Debian/GNU
+mtools package.
+
+.SH "SEE ALSO"
+.BR mtools (1), 
+.BR gzip (1),
+.BR tar (1),
+.BR uz (1).
--- a/tgz.1	1969-12-31 19:00:00.000000000 -0500
+++ b/tgz.1	2003-01-19 14:24:39.000000000 -0500
@@ -0,0 +1,56 @@
+.\" tgz.1
+.\"		(c) 2001 Filip Van Raemdonck <mechanix@debian.org>
+.\"
+.\" This manpage is free documentation.
+.\" Permission is granted to copy, distribute and/or modify this document
+.\" under the terms of the GNU Free Documentation License, Version 1.1 or any
+.\" later version published by the Free Software Foundation; with no Invariant
+.\" Sections, no Front-Cover Texts, and no Back Cover Texts.
+.\"
+.\" This  is distributed in the hope that it will be useful,
+.\" but WITHOUT ANY WARRANTY; without even the implied warranty of
+.\" MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+.\"
+.\" You should have received a copy of the GNU Free Documentation License
+.\" along with this program; if not, write to: 
+.\"		Free Software Foundation, Inc.
+.\"		59 Temple Place - Suite 330
+.\"		Boston, MA  02111-1307, USA.
+.\"
+.\"
+.ig
+A copy of the GNU Free Documentation License is available in the
+Debian package in the file /usr/share/doc/mtools/copyright.
+..
+.ig
+A copy of the GNU Free Documentation License is available in the 
+Debian source package in the file debian/copyright.
+..
+.TH TGZ "1" "May 2002" "mtools 3.9.8" "Mtools Users Manual"
+.SH NAME
+tgz \- makes a gzip'd tar archive
+.SH SYNOPSIS
+.B tgz
+[
+.I destination
+[
+.I source ...
+] ]
+.SH DESCRIPTION
+Make a gzip'd tar archive with the name of the first parameter out of specified
+files or, if no source files are specified, from everything in the current
+directory.
+If the first parameter is omitted as well, the archive will be written to
+stdout.
+.SH BUGS
+.B tgz
+requires gzip in the user's path.
+It also needs gnu tar or something close due to use of --exclude, --totals
+and -S.
+.SH AUTHOR
+Filip Van Raemdonck (mechanix@debian.org) wrote this page for the
+.I Debian/GNU
+mtools package.
+.SH "SEE ALSO"
+.BR gzip (1),
+.BR tar (1)
--- a/uz.1	1969-12-31 19:00:00.000000000 -0500
+++ b/uz.1	2003-01-19 14:24:39.000000000 -0500
@@ -0,0 +1,43 @@
+'\" t
+.\" ** The above line should force tbl to be a preprocessor **
+.\" Man page for uz
+.\"
+.\" Copyright (C), 2000, Robert A.R. King
+.\"
+.\" You may distribute under the terms of the GNU General Public
+.\" License as specified in the file COPYING that comes with the mtools
+.\" package
+.\"
+.\" Fri Dec  1 01:50:54 EST 2000 Robert A.R. King (Robert.King@mailbox.gu.edu.au)
+.\" 
+.TH UZ 1 "Wed Feb 23 00:00:00 EET 2000" "" "Mtools Users Manual"
+.SH NAME
+uz \- gunzips and extracts a gzip'd tar'd archive
+.SH SYNOPSIS
+.\" The command line
+.B uz 
+.I file
+.SH DESCRIPTION
+.B uz
+extracts a gzip'd tar'd archive, that is a \fBtar\fR(1) archive
+compressed with the \fBgzip\fR(1) utility.
+It is not strictly necessary on Debian GNU/Linux, because the GNU
+\fBtar\fR(1) program provides the same capability with the command
+
+.B tar -xzf
+.I file
+
+but this utility is provided in the mtools package for other platforms and 
+is retained here for completeness.
+
+.SH AUTHOR
+Robert King (Robert.King@mailbox.gu.edu.au) wrote this page for the
+.I Debian/GNU
+mtools package.
+
+
+.SH "SEE ALSO"
+.BR mtools (1), 
+.BR gzip (1),
+.BR tar (1),
+.BR lz (1).
--- a/mcomp.1	1969-12-31 19:00:00.000000000 -0500
+++ b/mcomp.1	2003-03-03 12:20:06.000000000 -0500
@@ -0,0 +1,44 @@
+'\" t
+.\" ** The above line should force tbl to be a preprocessor **
+.\" Man page for mcomp
+.\"
+.\" Copyright (C), 2003, Luis Bustamante
+.\"
+.\" You may distribute under the terms of the GNU General Public
+.\" License as specified in the file COPYING that comes with the mtools
+.\" package
+.\"
+.\" Mon Mar  3 11:58:15 COT 2003 Luis Bustamante <luferbu@fluidsignal.com>
+.\" 
+.TH MCOMP 1 "Mon Mar  3 11:58:15 COT 2003" "" "Mtools Users Manual"
+.SH NAME
+mcomp \- Compares two files using mtools
+.SH SYNOPSIS
+.\" The command line
+.B mcomp
+.I file-on-floppy
+.I local-file
+.SH DESCRIPTION
+.B mcomp
+compares two files, the first one must be on a floppy disk so it can
+be accesed via \fBmtools\fR.
+It is not strictly necessary on Debian GNU/Linux, because the diffutils
+\fBcmp\fR(1) program provides the same capability after copying the
+file locally with
+
+.B mcopy
+.I file
+.I destination
+
+but this utility is provided in the mtools package for other platforms and 
+is retained here for completeness.
+
+.SH AUTHOR
+Luis Bustamante <luferbu@fluidsignal.com> wrote this page for the
+.I Debian/GNU
+mtools package.
+
+.SH "SEE ALSO"
+.BR mtools (1),
+.BR cmp (1), 
+.BR mcopy (1)
--- a/mxtar.1	1969-12-31 19:00:00.000000000 -0500
+++ b/mxtar.1	2003-03-03 12:20:11.000000000 -0500
@@ -0,0 +1,48 @@
+'\" t
+.\" ** The above line should force tbl to be a preprocessor **
+.\" Man page for mxtar
+.\"
+.\" Copyright (C), 2003, Luis Bustamante
+.\"
+.\" You may distribute under the terms of the GNU General Public
+.\" License as specified in the file COPYING that comes with the mtools
+.\" package
+.\"
+.\" Mon Mar  3 11:58:15 COT 2003 Luis Bustamante <luferbu@fluidsignal.com>
+.\" 
+.TH MXTAR 1 "Mon Mar  3 11:58:15 COT 2003" "" "Mtools Users Manual"
+.SH NAME
+mxtar \- Wrapper for using GNU tar directly from a floppy disk
+.SH SYNOPSIS
+.\" The command line
+.B mxtar 
+[
+.B \-
+]
+.I [ tar-options ]
+.I file
+.SH DESCRIPTION
+.B mxtar
+let you use GNU \fBtar\fR(1) on a floppy disk without mounting it using
+\fBmtools\fR(1).
+It is not strictly necessary on Debian GNU/Linux, because the GNU
+\fBtar\fR(1) program provides the same capability after copying the
+file locally with
+
+.B mcopy
+.I file
+.I destination
+
+but this utility is provided in the mtools package for other platforms and 
+is retained here for completeness.
+
+.SH AUTHOR
+Luis Bustamante <luferbu@fluidsignal.com> wrote this page for the
+.I Debian/GNU
+mtools package.
+
+
+.SH "SEE ALSO"
+.BR mtools (1),
+.BR tar (1), 
+.BR mcopy (1)
--- a/mtools.1	2009-11-04 07:34:02.000000000 +1100
+++ b/mtools.1	2010-02-15 12:12:13.000000000 +1100
@@ -1,5 +1,5 @@
 '\" t
-.TH mtools.1 3 "03Nov09" mtools-4.0.12
+.TH mtools 1 "03Nov09" mtools-4.0.12
 .SH Name
 mtools - utilities to access DOS disks in Unix.
 '\" t
--- a/mtools.5	2009-11-04 07:34:02.000000000 +1100
+++ b/mtools.5	2010-02-15 12:12:13.000000000 +1100
@@ -1,5 +1,5 @@
 '\" t
-.TH mtools.1 3 "03Nov09" MTOOLS MTOOLS
+.TH mtools 5 "03Nov09" MTOOLS MTOOLS
 .SH Name
 mtools.conf - mtools configuration files
 '\" t
@@ -20,7 +20,7 @@ mtools.conf - mtools configuration files
 .SH Description
 .PP
 This manpage describes the configuration files for mtools. They 
-are called \fR\&\f(CW\(if/usr/local/etc/mtools.conf\(is\fR and \fR\&\f(CW\(if~/.mtoolsrc\(is\fR. If
+are called \fR\&\f(CW\(if/etc/mtools.conf\(is\fR and \fR\&\f(CW\(if~/.mtoolsrc\(is\fR. If
 the environmental variable \fR\&\f(CWMTOOLSRC\fR is set, its contents is used
 as the filename for a third configuration file. These configuration
 files describe the following items:
@@ -35,7 +35,7 @@ files describe the following items:
 .iX "c Name of configuration files"
 .iX "c Location of configuration files"
 .PP
-\&\fR\&\f(CW\(if/usr/local/etc/mtools.conf\(is\fR is the system-wide configuration file,
+\&\fR\&\f(CW\(if/etc/mtools.conf\(is\fR is the system-wide configuration file,
 and \fR\&\f(CW\(if~/.mtoolsrc\(is\fR is the user's private configuration file.
 .PP
 On some systems, the system-wide configuration file is called
@@ -456,7 +456,7 @@ implied, unless explicitly disabled by \
 Mtools uses its root privileges to open the device, and to issue the
 actual SCSI I/O calls.  Moreover, root privileges are only used for
 drives described in a system-wide configuration file such as
-\&\fR\&\f(CW\(if/usr/local/etc/mtools.conf\(is\fR, and not for those described in
+\&\fR\&\f(CW\(if/etc/mtools.conf\(is\fR, and not for those described in
 \&\fR\&\f(CW\(if~/.mtoolsrc\(is\fR or \fR\&\f(CW\(if$MTOOLSRC\(is\fR.  
 .TP
 \&\fR\&\f(CWprivileged\fR\ 
@@ -465,7 +465,7 @@ drives described in a system-wide config
 When set to 1, this instructs mtools to use its set-uid and set-gid
 privileges for opening the given drive.  This option is only valid for
 drives described in the system-wide configuration files (such as
-\&\fR\&\f(CW\(if/usr/local/etc/mtools.conf\(is\fR, not \fR\&\f(CW\(if~/.mtoolsrc\(is\fR or
+\&\fR\&\f(CW\(if/etc/mtools.conf\(is\fR, not \fR\&\f(CW\(if~/.mtoolsrc\(is\fR or
 \&\fR\&\f(CW\(if$MTOOLSRC\(is\fR).  Obviously, this option is also a no op if mtools is
 not installed setuid or setgid.  This option is implied by 'scsi=1', but
 again only for drives defined in system-wide configuration files.
@@ -590,7 +590,7 @@ The configuration files are parsed in th
 compiled-in defaults
 .TP
 2.\ 
-\&\fR\&\f(CW\(if/usr/local/etc/mtools.conf\(is\fR
+\&\fR\&\f(CW\(if/etc/mtools.conf\(is\fR
 .TP
 3.\ 
 \&\fR\&\f(CW\(if/etc/mtools\(is\fR
@@ -608,10 +608,10 @@ variable)
 Options described in the later files override those described in the
 earlier files. Drives defined in earlier files persist if they are not
 overridden in the later files. For instance, drives A and B may be
-defined in \fR\&\f(CW\(if/usr/local/etc/mtools.conf\(is\fR and drives C and D may be
+defined in \fR\&\f(CW\(if/etc/mtools.conf\(is\fR and drives C and D may be
 defined in \fR\&\f(CW\(if~/.mtoolsrc\(is\fR However, if \fR\&\f(CW\(if~/.mtoolsrc\(is\fR also
 defines drive A, this new description would override the description of
-drive A in \fR\&\f(CW\(if/usr/local/etc/mtools.conf\(is\fR instead of adding to it. If
+drive A in \fR\&\f(CW\(if/etc/mtools.conf\(is\fR instead of adding to it. If
 you want to add a new description to a drive already described in an
 earlier file, you need to use either the \fR\&\f(CW+drive\fR or \fR\&\f(CWdrive+\fR
 keyword.
